resources/assets/css/_forms.css

```diff
@@ -243,3 +243,14 @@
 .taggle {
     @apply space-x-2 bg-theme-primary-100;
 }
+
+.disabled-input .taggle > .close {
+    pointer-events: all;
+}
+
+.disabled-input .taggle_input,
```

Comment:
this is taggle specific as it expects a certain structure to change to a disabled state, name it accordingly